app/uk/gov/hmrc/agentservicesaccount/auth/AuthActions.scala

```diff
@@ -61,15 +61,15 @@ class AuthActions @Inject()(logger: LoggerLike,
             case Some(arn) =>
               body(AgentInfo(arn, credRole))
             case None =>
-              logger.warn("No AgentReferenceNumber found in HMRC-AS-AGENT enrolment - this should not happen. Denying access.")
-              Future successful Forbidden
+              logger.warn("No AgentReferenceNumber found in HMRC-AS-AGENT enrolment -- redirecting to /agent-subscription/start.")
```

Comment:
does this log message make sense? Is it that they don't have the HMRC-AS-AGENT enrolment at all.

Reply:
message has been updated